src/app/features/fee-editor/use-fee-editor.ts

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from 'react';
+
+import { isDefined } from '@leather.io/utils';
+
+import type { EditorFee, EditorFees } from './fee-editor.context';
+
+interface UseFeeEditorArgs {
+  editorFees: EditorFees;
+  getCustomEditorFee(rate: number): EditorFee;
+}
+export function useFeeEditor({ editorFees, getCustomEditorFee }: UseFeeEditorArgs) {
+  const [currentEditorFee, setCurrentEditorFee] = useState<EditorFee | null>(
```

Comment:
Looking at this setup now bc came from `useFees` (see deleted file). The tests are failing bc the fees stay in loading state. I had a console.log that was helping to trigger a rerender or this remained `null`. I think I'll have to use a loader component or something or we have to use `useEffects` to set state which like below, which isn't good.

Reply:
+1 to loaders or another kind of guard that prevents need for useEffect

Reply:
My last commit refactors to use loaders and removes the two hooks causing issues.